storage/payload.py

```diff
@@ -62,20 +62,33 @@ def chunk_lambda() -> Any:
     def get_by_lines(self, range_start: int) -> Iterator[tuple[bytes, int, int, Optional[int]]]:
         original_range_start: int = range_start
 
+        is_gzipped: bool = False
+        is_b64encoded: bool = False
         try:
             base64_decoded = base64.b64decode(self._payload, validate=True)
             if not base64_decoded.startswith(b"\037\213"):  # gzip compression method
                 base64_decoded.decode("utf-8")
+                is_b64encoded = True
+            else:
+                is_b64encoded = True
```

Comment:
should this be `False`?

Reply:
good catch!

Reply:
got it - in this case then it would be good to refactor this, it looks pretty weird `if foo: bar = True; else bar = False`...

Reply:
I will add comments to explain the behaviour for the moment and think about a way to refactor in the future